### PR TITLE
Problem: rocksdb release tar file naming not aligned with existing naming

### PR DESCRIPTION
**Pull request by allthatjazzleo**
Solution:
fix naming to align

👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [ ] Have you read the [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md)?
- [ ] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [ ] Have you rebased your work on top of the latest master? 
- [ ] Have you checked your code compiles? (`make`)
- [ ] Have you included tests for any non-trivial functionality?
- [ ] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-org-chain/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

### CODE REVIEW

**Comment by yihuang** [comment]
https://github.com/uraimo/run-on-arch-action#advanced-example
maybe we can use this to build for `aarch64`?

**Comment by allthatjazzleo** [comment]
> https://github.com/uraimo/run-on-arch-action#advanced-example maybe we can use this to build for `aarch64`?

sure I will test it

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -26,16 +26,21 @@ jobs:
           signingKey: "${{ secrets.CACHIX_SIGNING_KEY }}"
       - name: nix bundle
         run: |
+          MAINNET_TAR="cronos_$(echo ${GITHUB_REF##*/} | cut -c 2-)-rocksdb_Linux_x86_64.tar.gz"
```
Comment (JayT106) [inline .github/workflows/release.yml:29]: maybe we can parameterize the release filename like, 
```
MAINNET_TAR="cronos_$(echo ${GITHUB_REF##*/} | cut -c 2-)-${ db }_${ arch }.tar.gz"
```

File: Makefile [hunk 2]
```diff
@@ -11,10 +11,6 @@ TESTNET_FLAGS ?=
 
 VERSION := $(shell echo $(shell git describe --tags 2>/dev/null ) | sed 's/^v//')
 COMMIT := $(shell git log -1 --format='%H')
-NETWORK ?= mainnet
-COVERAGE ?= coverage.txt
-BUILDDIR ?= $(CURDIR)/build
-LEDGER_ENABLED ?= true
```
Comment (JayT106) [inline Makefile:17]: don't understand how these relate to the release file naming?

Comment (yihuang) [reply Makefile:17]: I think he just conveniently remove several duplicated lines, not related to the subject.

Comment (allthatjazzleo) [reply Makefile:17]: Yes, I conveniently removed several duplicated lines

File: .goreleaser.yml [hunk 3]
```diff
@@ -202,4 +202,4 @@ snapshot:
   name_template: "{{ .Tag }}-next"
 
 release:
-  draft: true
+  prerelease: true
```
Comment (yihuang) [inline .goreleaser.yml:205]: https://github.com/softprops/action-gh-release/issues/215

TODO: use back draft after the issue is fixed.

**Review by yihuang — APPROVED** [review]
LGTM

**Pull request merged**